src/bolus/index.js

```diff
@@ -0,0 +1,71 @@
+const match = require('../match');
+const { base } = require('../base/index');
+
+class BolusScreen extends base.Screen {
+    constructor(language) {
+        super({
+            screenText: language.bolusScreen,
+            generalText: language.general,
+            header: {
+                backLabel: language.general.Cancel,
+            },
+            open: {
+                isBtn: true,
+                label: language.bolusScreen.Header,
+            },
+        });
+    }
+    /**
+     * @override Screen.BackButton()
+     * */
+    BackButton() {
+        return match.accessible.ButtonBarButton(this.backLabel);
+    }
+    async Deliver() {
```

Comment:
[nit] Might want to consider separating out controls versus actions. Perhaps list all of the controls (e.g. "BackButton", "SaveAndDeliverButton") and then list the actions available (e.g. "Deliver", "SetBolusAmount).

Reply:
yeah - that would be nice